src/lib/loaders/__tests__/batchLoader.test.ts

```diff
@@ -82,6 +82,19 @@ describe("batchLoader", () => {
   })
 })
 
+describe("serializeParams", () => {
+  const { serializeParams } = require("../batchLoader")
```

Comment:
Btw, why are you loading these functions this way? If you would import them as normal, you could do `describe(serializeParams, () => …)` instead and have the name of the tests updated automatically if we’d ever rename the function (using the refactor tools).

Reply:
That's a good tip!

I'm doing the imports this way because earlier in the tests I mock the dependency to override the feature flag. 

https://github.com/artsy/metaphysics/blob/batching-bugfix/src/lib/loaders/__tests__/batchLoader.test.ts#L10

Reply:
Oh, but actually it doesn’t seem like you’d need to do any mocking for that, you can just change the value and reset it from an `afterEach`?

Anyways, not a blocker for this PR.